proto/cosmos/auth/v1beta1/auth.proto

```diff
@@ -36,6 +36,16 @@ message ModuleAccount {
   repeated string permissions  = 3;
 }
 
+// ModuleCredential represents a unclaimable pubkey for base accounts
+//
+// Since: cosmos-sdk 0.47
+message ModuleCredential {
+  string module_name = 1;
+  // derivation_keys is for deriving a module account
+  // adding more keys creates sub-accounts
```

Comment:
I would add comments that the 1st item is passed into `Module`, and the rest into `Derive`